apps/privacy-panel/js/tc/applications.js

```diff
@@ -44,7 +46,13 @@ function(panels, appList, appDetails) {
       // some apps might have a localized name in their manifest
       window.addEventListener('localized', refreshAppList);
 
-      appList.init(permissionTable).then(this.renderAppList.bind(this),
+      var appSearch = document.getElementById('tc-appSearch');
+      appSearch.onsubmit = () => this._closeSearchBox();
```

Comment:
Actually, that’s the standard from the Building Blocks: the “clear” button is a `submit` one.